OrderApi/Migrations/20180816230225_default.Designer.cs

```diff
@@ -0,0 +1,85 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using ShoesOnContainers.Services.OrderApi.Data;
+
+namespace OrderApi.Migrations
+{
+    [DbContext(typeof(OrdersContext))]
+    [Migration("20180816230225_default")]
+    partial class @default
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "2.1.1-rtm-30846")
+                .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+            modelBuilder.Entity("ShoesOnContainers.Services.OrderApi.Models.Order", b =>
+                {
+                    b.Property<int>("OrderId")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<string>("Address");
+
+                    b.Property<string>("BuyerId");
+
+                    b.Property<string>("FirstName");
+
+                    b.Property<string>("LastName");
+
+                    b.Property<DateTime>("OrderDate");
+
+                    b.Property<int>("OrderStatus");
+
+                    b.Property<decimal>("OrderTotal");
+
+                    b.Property<string>("PaymentAuthCode");
+
+                    b.Property<string>("UserName");
+
+                    b.HasKey("OrderId");
+
+                    b.ToTable("Orders");
+                });
+
+            modelBuilder.Entity("ShoesOnContainers.Services.OrderApi.Models.OrderItem", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd();
+
+                    b.Property<int>("OrderId");
+
+                    b.Property<string>("PictureUrl");
+
+                    b.Property<int>("ProductId");
```

Comment:
ProductId and ProductName here would need to be changed to EventId and EventName if planning to change those property names to match the other services